report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
+Our goal is to test each of the components of the pipeline individually rather than trying to reproduce the results to absolute perfection.
+
+Similarly, in our replica of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citet{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for replication without unsubstantiated assumptions.
+They mention the use of Median Absolute Deviation for this purpose, but the specifics of their approach remain unclear.
+Our decision to omit outlier detection in the ICA phase of our pipeline is also influenced by a preference for retaining a more comprehensive dataset, despite the potential inclusion of outliers.
+Further supporting this decision is input from one of the original authors involved in the ChemCam project.
+This author highlighted the extensive efforts invested in developing the ChemCam calibration dataset, suggesting a low presence of significant outliers.
+While it would be ideal to include outlier detection in the ICA phase to ensure alignment with the original pipeline, implementing it based on substantial assumptions would be counterproductive, potentially compromising the integrity of our analysis.
+
+After examining the results in section \ref{sec:results}, we will discuss the implications of these design choices in section \ref{sec:discussion}.
+
+\subsubsection{Joint Approximate Diagonalization of Eigenmatrices (JADE) and Regression}
+After preprocessing the data, we are ready to perform ICA.
+This is done using the Joint Approximate Diagonalization of Eigenmatrices (JADE) algorithm, which is used to calculate the mixing matrix.
+This mixing matrix is an $N \times M$ matrix, where $N$ is the number of samples and $M$ is the number of independent components.
+By taking the product of the mixing matrix and the normalized data, we get the estimated sources.
+This new matrix is an $M \times P$ matrix, where $M$ is the number of independent components, and $P$ is the number of wavelengths.
+
+After performing ICA, we post-process by computing the correlation between the independent components and the wavelengths.
+Using this, we can identify which wavelengths are associated with which independent components by computing the maximum correlation for each wavelength.
+This gives us a matrix of ICA scores that we then use for regression.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.45\textwidth]{images/ica_regression.png}
+	\caption{Regression in the ICA phase of our recreation of the pipeline.}
+	\label{fig:ica_regression}
+\end{figure}
+
+The regression process is illustrated in figure \ref{fig:ica_regression}.
+Since we are implementing this recreation in Python, a different programming language than the original pipeline, we cannot use the same libraries for regression.
+We use the \texttt{LinearRegression} class from the \texttt{scikit-learn} library\cite{scikit-learn}.
```

Comment:
Doesn't seem necessary to write. Do they even specify which libraries they use?
IMO remove.

Reply:
Agreed and removed.